packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -26,9 +26,13 @@ const AccessoryAlignments = {
 
 const propTypes = {
   /**
-   * The column layout in which to present the displays. One of `oneColumn`, `twoColumn`.
+   * The visual column layout in which to present the displays. One of `oneColumn`, `twoColumn`.
    */
   layout: PropTypes.oneOf(['oneColumn', 'twoColumns']),
+  /**
+  * Determines whether the displays are programmatically separated by row or as true columns when layout is set to `twoColumns`.
+  */
```

Comment:
If the value of `trueColumn` affects screenreader behavior, can we add that to this description?

Reply:
sure thing: [ec3f4ef](https://github.com/cerner/terra-clinical/pull/912/commits/ec3f4efe61e2a311b363a0ffb3f2e0fd079cd9bc)